### ISSUE TITLE
highlightBlock with specified language

### ISSUE THREAD

**Issue by sf-wind** [isagalaev/highlight.js#902]
Hello,

I'd like to invoke hljs.highlightBlock in javascript on a DOM element. However, the auto-detected language is incorrect, so I need to specify a language. But it seems the method doesn't take a second argument for the language. 

Is it possible to choose a language when highlighting in javascript (not through class)?

Thanks.

**Comment by isagalaev** [comment]
You can still set a class for the DOM element before calling `hljs.highlightBlock`. Or, going the other way, you could first get textual content of the element, call `hljs.highlight` for it and then assign the result to `element.innerHTML`.

**Comment by renfredxh** [comment]
I ran into this problem earlier and was struggling to figure out to get `highlightBlock` to work with a specific language until I came across this issue. Setting the class with the language I wanted solved the issue. 

Is that feature documented anywhere? It might be helpful to note this in the [docs for highlightBlock](https://highlightjs.readthedocs.org/en/latest/api.html#highlightblock-block) for future people that might run into this.
